### PR TITLE
coffee: add page

### PR DESCRIPTION
**Pull request by idleberg**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/coffee.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run script:
```
Comment (sbrl) [inline pages/common/coffee.md]: Perhaps `Run a script` would read better?

File: pages/common/coffee.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`coffee {{filename}}.coffee`
+
+- Compile to JavaScript and save file:
```
Comment (sbrl) [inline pages/common/coffee.md]: Perhaps ....`and save to a file` would read better?

File: pages/common/coffee.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`coffee --compile {{filename}}.coffee`
+
+- Compile to JavaScript and save as file:
```
Comment (sbrl) [inline pages/common/coffee.md]: Perhaps ...`and output to a specific file` would read better?

Also, could `{{filename}}` be `{{path/to/filename.js}}` perhaps? That would make it more consistent with the existing tldr-pages we have here.

File: pages/common/coffee.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`coffee --compile {{filename}}.coffee --output {{filename}}`
+
+- Run interactive REPL:
+
+`coffee --interactive`
+
+- Watch script for changes and rerun:
```
Comment (sbrl) [inline pages/common/coffee.md]: What is meant by rerun here?

Comment (idleberg) [reply pages/common/coffee.md]: What this does is 1. run the script, 2. watch for changes and 3. run again if it changed.

File: pages/common/coffee.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run a script:
+
+`coffee {{path/to/filename}}.coffee`
+
+- Compile to JavaScript and save to a file:
```
Comment (agnivade) [inline pages/common/coffee.md]: We should clarify that it saves to a file with the same name.

File: pages/common/coffee.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run a script:
+
+`coffee {{path/to/filename}}.coffee`
```
Comment (agnivade) [inline pages/common/coffee.md]: Use {{path/to/file}} everywhere. I think that is more idiomatic with the rest of the pages.

File: pages/common/coffee.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+- Compile to JavaScript and save to a file:
+
+`coffee --compile {{path/to/filename}}.coffee`
+
+- Compile to JavaScript and save as file:
```
Comment (agnivade) [inline pages/common/coffee.md]: "and save to a given output file"

**Comment by agnivade** [comment]
Whoops, thanks @idleberg for the correction !

File: pages/common/coffee.md (hunk 3 above)
Comment (sbrl) [reply pages/common/coffee.md]: Ah, I see

**Pull request merged**